File: Praktikum 6/c/PRAK604_2410817110012_GtMuhammadLuthfiKamil.c
#include<stdio.h>
#include <string.h>
int  main(){
    char asli[100];
    char palsu[100];
    scanf("%[^\n]%*c", &asli);
    scanf("%[^\n]%*c", &palsu);
    int code, pesan, x, i=0, j=0;
    code = strlen(asli);
    pesan = strlen(palsu);
    if(code != pesan){
        printf("Panjang kalimat berbeda, pesan palsu");
    }else{
        for(x = 0; x < code; x++){
            if(asli[x]==palsu[x]){
                printf("*");
                i++;
            }else{
                printf("#");
                j++;
            }    
        }
        printf("\n* = %d", i);
        printf("\n# = %d", j);
    if(i >= j){
        printf("\nPesan Asli");
    }else if(i < j){
        printf("\nPesan Palsu");
    }
    }
}

File: Praktikum 6/py/PRAK604_2410817110012_GtMuhammadLuthfiKamil.py
kode = input().rstrip()
pesan = input().rstrip()

jumlahBintang = 0
jumlahPagar = 0

pesanTransformasi = ""
for i in range(len(kode)):
    if kode[i] == pesan[i]:
        pesanTransformasi += '*'
        jumlahBintang += 1
    else:
        pesanTransformasi += '#'
        jumlahPagar += 1

if len(kode) != len(pesan):
    print("Panjang kalimat berbeda, pesan palsu")
else:
    print(pesanTransformasi)
    print("* =", jumlahBintang)
    print("# =", jumlahPagar)

    if jumlahBintang >= jumlahPagar:
        print("Pesan Asli")
    else:
        print("Pesan palsu")